docs/examples.md

```diff
@@ -41,6 +41,18 @@ check_android_task:
 !!! info
     Please don't forget to setup [Remote Build Cache](#build-cache) for your Gradle project. Or at least [simple folder caching](#gradle-caching).
 
+## Apache Maven
+
+Here is an example `.cirrus.yml` that builds a Maven project (and runs basic tests on code to make sure there are no simple errors):
+```yaml
+task:
+  name: Cirrus CI
+  container:
+    image: maven:3-jdk-8
```

Comment:
JDK 11 is actually the default one. You can probably change the tag to `maven:3` which is `maven:3-jdk11`.